src/main/java/teammates/common/util/Const.java

```diff
@@ -386,7 +386,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_QUESTION_GETLINK =
                 "Get a submission link to this particular question. "
                 + "Useful if you want students to answer individual questions separately or at different points in time.";
-        public static final String FEEDBACK_QUESTION_CANCEL = "Discard your changes";
+        public static final String FEEDBACK_QUESTION_DISCARDCHANGES = "Discard your changes";
```

Comment:
Reading several articles on this, it seems that if standardizing on a single type of pronoun (ie. all 'you' or all 'I'), using 'you' would be better. In the case of status messages, it directly addresses the user if any suggestions are made (eg. "Please check that your details are correct"), which is more natural sounding than the alternative (Please check that my details are correct"). In addition, rewriting sentences from 'I' -> 'You' would be easier than the alternative such as:

"I want to manually publish the results." -> "Check this if you want to manually publish the results"

as opposed to

"Please check that you have filled in all fields" -> ???

Most articles ([link 1](https://ustwo.com/blog/mine-or-yours-a-ux-geeks-thoughts/)) ([link 2](https://dcurt.is/yours-vs-mine)) advocate the use of 'You' exclusively, while [Microsoft](https://msdn.microsoft.com/en-us/library/dn742477.aspx#person) suggests a mix of both depending on the context.